tfx/components/trainer/executor_test.py

```diff
@@ -47,20 +47,30 @@ def setUp(self):
         self._testMethodName)
 
     # Create input dict.
-    examples = standard_artifacts.Examples()
-    examples.uri = os.path.join(self._source_data_dir,
+    e1 = standard_artifacts.Examples()
+    e1.uri = os.path.join(self._source_data_dir,
                                 'transform/transformed_examples')
-    examples.split_names = artifact_utils.encode_split_names(['train', 'eval'])
+    e1.split_names = artifact_utils.encode_split_names(['train', 'eval'])
+
+    e2 = standard_artifacts.Examples()
+    e2.uri = os.path.join(self._source_data_dir,
+                                'transform/transformed_examples')
+    e2.split_names = artifact_utils.encode_split_names(['train', 'eval'])
```

Comment:
wondering can we just do a copy instead of duplicated code

Reply:
I wasn't sure if shallow copying is enough to test our multiple artifact (i.e. just using python assignment), which is why I decided on this way for now. is there some tfx utility for copying artifacts?

Reply:
we can use proto `CopyFrom()`. See [this](https://developers.google.com/protocol-buffers/docs/reference/python-generated) and search for 'CopyFrom'.

Reply:
spoke with Ruoyu, decided to use `copy.deepcopy()` instead.